### DISCUSSION TITLE
Send mails using nodemailer across Mailu mail server issue (Rejected (anti-spoofing: DMARC compliance is enforced for local domains, regardless of the policy setting))

### DISCUSSION THREAD

**Discussion by MahmoudSamy201** [Mailu/Mailu#3639] [User support]
I setup mailu mail server which is working on default port 25 and Iam also have node api running in docker container and i want to send mails from my node api using nodemailer library which use mail.mydomain & port 25 but I got this error when trying to send mails so how can I solve this issue 

8CD677239: milter-reject: END-OF-MESSAGE from unknown[172.19.0.1]: 5.7.1 Rejected (anti-spoofing: DMARC compliance is enforced for local domains, regardless of the policy setting); from=<support@sender-domain.com> to=<recipiant@gmail.com> proto=ESMTP helo=<[127.0.0.1]>

**Comment by pini-gh** [comment]
Hi,
I do have this issue as well, for a local address which has the "allow-spoofing" setting. Is there a workaround?
